docs/Account/methods/userOpMethods.md

```diff
@@ -208,8 +209,17 @@ type UserOpReceipt = {
 }
 
 ```
+You can use `waitForTxHash` to get the userOpHash or transactionHash, without waiting for the transaction to be mined.
```

Comment:
waitForTxHash gives you only transactionHash and status 

```
{
  state: 'SUBMITTED',
  transactionHash: '0xa76b86287d316454a25ed1ea0540bccc11b6aadce5f76a54a7713a30a7b4f1be'
}
```

when you call sendUserOp you receive userOpHash at that stage only.